### ISSUE TITLE
Spellcheck: Ignore words that start with a number

### ISSUE THREAD

**Issue by metal450** [giuspen/cherrytree#1052]
There are some cases in which "words" may begin with a number - i.e. "24hr", "5min", "30sec."  In CT (C++ version), each of these are underlined in red by the spellchecker.  I believe standard practice is for to ignore any word (from the spellchecker) which begins with a number (i.e. it's how it works in Firefox, OneNote, Word, etc) :)

**Comment by txe** [comment]
I don't think we can do it, we use gspell and it doesn't have any options to adjust its behavior.

**Comment by metal450** [comment]
Bummer, but maybe if u just keep it in the back of ur mind, you'll stumble on some unexpected/easy solution at some point.  Like with indentation :)

**Comment by metal450** [comment]
Related, so I'll add it to this same issue: another common thing that spellcheckers ignore are proper nouns.  I'm not sure exactly how this works - maybe it's just they don't spellcheck capitalized words? - but I've noticed that CT's spellchecker 'underlines' significantly more words than other text editors, mostly due to these proper nouns (i.e. people names, place names, business names, etc).  I would say that ~90% of the reported spelling errors are actually either proper nouns or words starting in numbers, which sort of drowns out the "true" errors.  Would be awesome if these too could be omitted :)

**Comment by metal450** [comment]
One more issue: it underlines every URL as a spelling error.
